tests/pytests/test_certifier_framework.py

```diff
@@ -497,33 +503,39 @@ def cc_trust_manager_get_certified(cctd):
 
     result = cctd.init_policy_key(cert_bin)
     assert result is True
-    print(' ... cctd.init_policy_key() succeeded.')
+    print(' ... cctd.init_policy_key() succeeded. Initialized serialized_policy_cert_ ')
 
-    # Open hard-coded key / platform endorsement & app-measurement files
-    attest_key_file_name             = CertPyTestsDir + '/data/attest_key_file.bin'
-    attest_endorsement_file_name = CertPyTestsDir + '/data/platform_attest_endorsement.bin'
-    example_app_measurement_file_name         = CertPyTestsDir + '/data/example_app.measurement'
+    # Another attempted fix was to read using this interface, which will read-in
+    # data as string, with special-case handling of Unicode surrogate chars.
+    # This did not work in all cases. Some tests still failed; in some cases
+    # we ended up reading incomplete data.
+    # with open(attest_key_file_name, encoding="utf-8", errors="surrogateescape") as attest_key_file:
 
+    # Open hard-coded key / platform endorsement & app-measurement files
+    # and read-in data as a bytestream.
+    attest_key_file_name = CertPyTestsDir + '/data/attest_key_file.bin'
     with open(attest_key_file_name, 'rb') as attest_key_file:
         attest_key_bin = attest_key_file.read()
 
+    example_app_measurement_file_name = CertPyTestsDir + '/data/example_app.measurement'
     with open(example_app_measurement_file_name, 'rb') as example_app_measurement_file:
         example_app_measurement = example_app_measurement_file.read()
 
+    attest_endorsement_file_name      = CertPyTestsDir + '/data/platform_attest_endorsement.bin'
     with open(attest_endorsement_file_name, 'rb') as attest_endorsement_file:
         platform_attest_endorsement_bin = attest_endorsement_file.read()
 
-    result = cctd.initialize_simulated_enclave(str(attest_key_bin),
-                                               str(example_app_measurement),
-                                               str(platform_attest_endorsement_bin))
+    result = cctd.initialize_simulated_enclave(attest_key_bin,
+                                               example_app_measurement,
+                                               platform_attest_endorsement_bin)
```

Comment:
This call invokes the interface using byte-streams that are established above using `read()` with `open(... rb)`.